qa-anvil.planx-pla.net/portal/gitops.json

```diff
@@ -151,30 +151,33 @@
         }
       ]
     },
-    "projectId": "search",
-    "graphqlField": "case",
-    "index": "",
+    "buttons": [
+      {
+        "enabled": false,
+        "type": "data",
+        "title": "Download Clinical",
+        "leftIcon": "user",
+        "rightIcon": "download",
+        "fileName": "clinical.json",
+        "dropdownId": "download"
+      },
+      {
+        "enabled": false,
+        "type": "manifest",
+        "title": "Download Manifest",
+        "leftIcon": "datafile",
+        "rightIcon": "download",
+        "fileName": "manifest.json",
+        "dropdownId": "download"
+      }
+    ],
     "table": {
-      "buttons": [
-        {
-          "enabled": false,
-          "type": "data",
-          "title": "Download Clinical",
-          "leftIcon": "user",
-          "rightIcon": "download",
-          "fileName": "clinical.json",
-          "dropdownId": "download"
-        },
-        {
-          "enabled": false,
-          "type": "manifest",
-          "title": "Download Manifest",
-          "leftIcon": "datafile",
-          "rightIcon": "download",
-          "fileName": "manifest.json",
-          "dropdownId": "download"
-        }
-      ]
+      "enabled": true
+    },
+    "arrangerConfig": {
+      "projectId": "search",
+      "graphqlField": "case",
+      "index": ""
```

Comment:
add "nodeCountField": "node_id" to the "arrangerConfig" blob

Reply:
I have made the necessary changes.